Gruntfile.js

```diff
@@ -62,7 +62,7 @@ module.exports = function( grunt ) {
 			safari: [ "bs_safari-4.0", "bs_safari-5.0", "bs_safari-5.1" ],
 			ie: [ "bs_ie-7", "bs_ie-8" ],
 			opera: [ "bs_opera-11.6", "bs_opera-12.16" ],
-			android: [ "bs_android-2.3" ]
+			android: [ "bs_android-4.4" ]
```

Comment:
Could you add a comment above this line that older Android devices are too unreliable to be included in CI (and finish that with `see #314` as was done below)?